EventsExpress/ClientApp/src/components/event/event-form.js

```diff
@@ -110,6 +115,7 @@ class EventForm extends Component {
                             <Field
                                 name='dateFrom'
                                 label='From'
+                                disabled={disabledDate ? true: false}
```

Comment:
It is already a boolean value, donot use ternary operator.

Reply:
Ok. I'll fix this in the next commit